docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -46,12 +44,11 @@ To use the existing {OpenStackShort} deployment password:
 ----
 ADMIN_PASSWORD=$(cat ~/tripleo-standalone-passwords.yaml | grep ' AdminPassword:' | awk -F ': ' '{ print $2; }')
 ----
-* Set service password variables to match the original deployment.
+* Set the service password variables to match the original deployment.
 Database passwords can differ in the control plane environment, but
-synchronizing the service account passwords is a required step.
+synchronizing the service account passwords is required.
 +
-For example, in developer environments with {OpenStackPreviousInstaller} Standalone, the
-passwords can be extracted like this:
+For example, in developer environments with {OpenStackPreviousInstaller} Standalone, the passwords can be extracted:
 +
 ----
 AODH_PASSWORD=$(cat ~/tripleo-standalone-passwords.yaml | grep ' AodhPassword:' | awk -F ': ' '{ print $2; }')
```

Comment:
I may send another PR to amend this example for better downstream suitability and remove the replica numbers below but i think this PR is good to merge.

Reply:
I created a bug so that i don't forget about it https://issues.redhat.com/browse/OSPRH-8880